src/components/cards/challenge/Challenge.tsx

```diff
@@ -25,81 +24,69 @@ interface ChallengeCardProps {
 }
 export default function ChallengeCard({ data, community, isCourseEnd }: ChallengeCardProps) {
   const { t } = useTranslation();
-
   const link = `/communities/${community.slug}/challenges/${data.id}`;
   const expiresAt = useMemo(() => (data.expiresAt ? new Date(data.expiresAt).toLocaleDateString() : null), [data.expiresAt]);
   const reward = isCourseEnd ? data?.rewards?.find((reward) => reward.type === "SUBMISSION") : data?.reward;
-  const totalReward = data?.rewards?.reduce((acc, reward) => (acc += Number(reward.amount)), 0);
+  const learningModulesCount = useMemo(() => (data?.learningModules?.length ? data?.learningModules.length : 0), [data?.learningModules?.length]);
```

Comment:
We don't have a useMemo here, and we don't need a ternary operator

you can just do `const learningModulesCount = data?.learningModules?.length || 0`

Reply:
@mhenriette can you make these changes and fix the conflicts